src/frontend/src/pages/PPQFormPage.tsx

```diff
@@ -0,0 +1,276 @@
+                      <option value={option}> {option}</option>
+                    ))}
+                  </select>
+                </div>
+              </div>
+            </div>
+
+            <div className="ppq-form-content">
+              {' '}
+              <h2>3. Personal information</h2>
+              <h3>Is personal information involved in your initiative?</h3>
+              <span>
+                Personal information is any recorded information about an
+                identifiable individual, other than business contact
+                information. Personal information includes information that can
+                be used to identify an individual through association or
+                reference.
+              </span>
+              <div>
+                <label>
+                  <input type="radio" value="yes" checked={true} />
+                  Yes
+                </label>
+              </div>
+              <div className="radio">
+                <label>
+                  <input type="radio" value="no" />
+                  No
+                </label>
+              </div>
+              <div className="radio">
+                <label>
+                  <input type="radio" value="notsure" />
+                  I'm not sure
+                </label>
+              </div>
+            </div>
+            <div className="ppq-form-content">
+              <h3>4. Other factors</h3>
+              <span>
+                Does your initiative involve any of the following? Check all
+                that apply.
+              </span>
+              <div>
+                {OtherFactor.map((name, index) => {
+                  return (
+                    <div>
+                      <div>
+                        <input
+                          type="checkbox"
+                          id={`custom-checkbox-${index}`}
+                          name={name}
+                          value={name}
+                        />
+                        <label htmlFor={`custom-checkbox-${index}`}>
+                          {name}
+                        </label>
+                      </div>
+                    </div>
+                  );
+                })}
+              </div>
+            </div>
+
+            <div className="ppq-form-content">
+              {' '}
+              <h2>5. Start date </h2>
+              <div className="row">
+                <div className="form-group col-md-6">
+                  <span>
+                    Do you have a proposed go-live or start date for the
+                    initative?
+                  </span>
+                  <div>
+                    <label>
+                      <input type="radio" value="yes" checked={true} />
+                      Yes
+                    </label>
+                  </div>
+                  <div className="radio">
+                    <label>
+                      <input type="radio" value="no" />
```

Comment:
Please refer to the enum values for them. May be work on them as you are integrating the API `POST /api/ppq`. Happy if you update it as part of the integration ticket/PR. This applies to all values.

Reply:
sure